### PR TITLE
Document more of the API

### PR DESCRIPTION
**Pull request by toolness**
* Adds documentation for the histogram results, fixing #69.
* Adds documentation for the general result format (e.g. pagination details).

## Notes

* I didn't document the contract details very well, partly because they're very similar to the query arguments, and I didn't want to violate DRY. I think we can improve upon this in a later PR.

## To do

- [x] Add documentation for Contract result fields.
- [x] Add documentation for autocomplete result fields.

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -304,6 +304,53 @@ def quantize(num, precision=2):
...
+            contains the labor rate.
+        * `vendor_name` is the name of the vendor that the
+            labor rate's contract is under.
+        * `education_level` is the minimum level of education
+            for the labor rate.
+        * `min_years_experience` is the minimum years of experience
+            for the labor rate.
+        * `hourly_rate_year1`, `current_price`, `next_year_price`,
+            and `second_year_price` contain pricing information for
+            the labor rate.
```
Comment (toolness) [inline api/views.py]: Uhhh, I forget what the differences between all these are, so I just lumped them together and provided a vague description. 🙁

Comment (hbillings) [reply api/views.py]: I *think* what's going on here is that `hourly_rate_year1` is the rate for the year the contract started;  `current_price` is the price based on the escalation rate, if applicable; and `next_year_price` and `second_year_price` pertain to out-year pricing (the +1/+2 buttons in the filtering system).

File: api/views.py [hunk 2]
```diff
@@ -304,6 +304,53 @@ def quantize(num, precision=2):
...
+    * `results` is an array containing the results for the
+    current page. Each item in the array contains the following keys:
+        * `id` is the internal ID of the rate in the CALC database.
+            This can be passed to the `exclude` query parameter to
+            exclude individual rates from search results.
+        * `idv_piid` is the contract number of the contract that
+            contains the labor rate.
+        * `vendor_name` is the name of the vendor that the
+            labor rate's contract is under.
+        * `education_level` is the minimum level of education
```
Comment (hbillings) [inline api/views.py]: This is different than the way the sort uses it, right?

Comment (toolness) [reply api/views.py]: Umm how do you mean?

**Pull request merged**